modules/virt-hot-plugging-memory.adoc

```diff
@@ -20,6 +20,10 @@ The system applies these changes immediately. If the VM is migratable, a live mi
 
 [NOTE]
 ====
-* Linux guests require a kernel version of 5.16 or later and Windows guests require the latest `viomem` drivers.
-* By default, you can hot-plug a VM with up to 4 times its initial CPU or memory configuration. For example, a VM with 1 vCPU and 2 GiB memory can scale up to 4 vCPUs and 8 GiB before a restart is needed. You can adjust this behavior cluster-wide by using the `maxHotplugRatio` parameter or override it in your VM.
```

Comment:
Did you mean to remove this line? If it is still relevant in 4.18, it should stay.